admin/openwisp-config/Makefile

```diff
@@ -1,14 +1,14 @@
-# openwisp.org
+# SPDX-Identifier-License: GPL-3.0-only
 #
-# This is free software, licensed under the GNU General Public License v3.
-# See /LICENSE for more information.
+# openwisp.org
+
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openwisp-config
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
No need to bump this. You just touch SPDX License Identifier and for that you don't want to really wear the flash memory on the routers.